raster/r.what/main.c

```diff
@@ -18,7 +18,7 @@
  *               for details.
  *
  *****************************************************************************/
-#define NFILES 400
+#define NFILES 1000
```

Comment:
How about 1024 (I think a few hyperspectral cameras offering that).

And the "TODO 
    Fix 400 maps limit" needs to be removed from the HTML manual page file.

Reply:
@neteler Sounds reasonable to me. 

I would not be even afraid of setting it to 2048 to be prepared for the next step.